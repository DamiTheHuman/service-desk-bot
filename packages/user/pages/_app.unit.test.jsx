import React from 'react';
import {expect, describe, it} from 'vitest';
import {render} from '@testing-library/react';
import App from './_app';
import Page from '.';

describe('Index Page', () => {
  it('renders correctly', () => {
    const {asFragment} = render(<App Component={Page} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
