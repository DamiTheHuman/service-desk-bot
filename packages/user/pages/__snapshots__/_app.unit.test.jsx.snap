// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Index Page > renders correctly 1`] = `
<DocumentFragment>
  <div
    class="index container p-4"
  >
    <div
      class="d-flex justify-content-between flex-column"
    >
      <h2>
        Service Desk Bot 
        <span
          class="h4 text-black-50"
        >
          v.1.0.0
        </span>
      </h2>
      <div
        class="mt-auto fixed-bottom container pb-4"
      >
        <form
          role="deliver-message"
        >
          <div
            class="input-group"
          >
            <span
              class="input-group-text"
              id="basic-addon1"
            >
              <i
                class="bi bi-person-circle"
              />
            </span>
            <input
              class="message-prompt form-control form-control-lg"
              id="message-prompt"
              name="message"
              placeholder="Message Service Desk Bot..."
              type="text"
              value=""
            />
          </div>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
